### ISSUE TITLE
Overfitting model

### ISSUE THREAD

**Issue by Nicho256** [Nicho256/Cancer_Classification#3]
The current VGG16 model is overfitting. The exact solution to overfitting is unknow. However, suggested ways to reduce the overfitting include:

1. Freezing the first layers of the pre-trained model.
2. Lowering the learning rate of the pre-trained model.
3. Using early stopping.
4. Making the model simpler by removing some layers.
5. Using L2 Regularization.
